### DISCUSSION TITLE
Keep getting a useClient error? anyone seen this.

### DISCUSSION THREAD

**Discussion by dev-cz** [wevm/wagmi#640] [General]
Using react nextjs for build out. Have various different components. Modal is created but when trying to build I get an error "Error: `useClient` must be used within `WagmiConfig`."  No documentation on this or how I could fix it either. Any help would be appreciated.

**Comment by tmm** [comment]
Can you link a GitHub repo or CodeSandbox with a reproduction?

**Reply by dev-cz** [reply]
Fixed it. Had to move my wrapped config up 1 line! Working now!. Thank you.
